Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -97,7 +97,7 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
 
         public static IEnumerable<object[]> GetClientInterfaces()
         {
-            return typeof(IEventsClient)
+            return typeof(GitHubClientTests)
```

Comment:
@shiftkey nice spot, I think that usage of interface instead of implementation would be appropriate
here.